src/util.cpp

```diff
@@ -488,6 +488,12 @@ namespace Sass {
 
       int cp = utf8::next(it, end);
 
+      // in case of \r, check if the next in sequence
+      // is \n and then advance the iterator.
+      if (cp == '\r' && it < end && utf8::peek_next(it, end) == '\n') {
```

Comment:
On second though, `\r` is a legal character in Unix and Windows. Now I wonder if Ruby Sass is doing the right thing in first place. :bulb: